content/en/docs/use-cases/gh-actions-app-builder.md

```diff
@@ -0,0 +1,219 @@
+---
+title: GitHub Actions Basic App Builder
+linkTitle: GitHub Actions Basic App Builder
+description: "How to build and push image tags for Flux from Git branches and tags."
+weight: 39
+---
+
+This guide shows how to configure GitHub Actions to build an image for each new commit pushed on a branch, for PRs, or for tags in the most basic way that Flux's automation can work with and making some considerations for both dev and production.
+
+In the [GitHub Actions Manifest Generation] guide, we saw some workflows that render manifests in CI and commit them back to Git. From [Image Update Guide] we saw that Flux can set `.spec.git.push.branch` to [Push updates to a different branch] than the one used for checkout. Those advanced techniques are above and beyond what is needed to work with Flux.
```

Comment:
If we send people from here to a guide that is 10x the size of this, we'll lose them. I think we should remove this from here, and put it down below at other links.

Reply:
I agree, I have made this change in [428cdad](https://github.com/fluxcd/website/pull/1011/commits/428cdadebdfbe64207521ed734a384e0c61f7310) – manifest generation is still mentioned, but less prominently and at the end.

The reference is now listed as part of a "Further Reading" section at the bottom that also provides some closure about how the reader should follow the Flux docs to make use of these built images from CI.

I added another sub-heading for caching, it provides no specific guidance other than fundamental "fast builds are better than slow ones" and only references other documentation to provide specifics so should hopefully remain evergreen, even if there are changes in the ecosystem.